.jenkins/build_docker_images.sh

```diff
@@ -3,20 +3,22 @@
 # Set variable to allow parallel building in the Docker image creation
 export DOCKER_BUILDKIT=1
 export BUILDKIT_PROGRESS=plain
-targets="build build_serialize_gt4py_dev"
-tags="hpc hpc-serialize"
 
 # Speed-up the compilations by using pre-built MPI, FMS, and ESMF images
 export BUILD_FROM_INTERMEDIATE=y
 
+# We build and test Docker images without CUDA support
+export CUDA=n
+
 # Build FV3 without and with Serialbox support enabled
 make pull_deps
-COMPILED_TAG_NAME=hpc make ${targets} 
+make build build_serialize_gt4py_dev 
```

Comment:
After some discussion, probably best to use `build_serialize` here and we need to find a more generic solution for GT4PY_DEV.